src/openms/include/OpenMS/CHEMISTRY/ISOTOPEDISTRIBUTION/CoarseIsotopePatternGenerator.h

```diff
@@ -93,21 +93,17 @@ namespace OpenMS
     * See also method run()
     **/
 
-  class OPENMS_DLLAPI CoarseIsotopePatternGenerator 
+  class OPENMS_DLLAPI CoarseIsotopePatternGenerator
     : public IsotopePatternGenerator
   {
 
  public:
-    CoarseIsotopePatternGenerator();
-
-    CoarseIsotopePatternGenerator(const Size& max_isotope);
-
-    CoarseIsotopePatternGenerator(const Size& max_isotope, const bool round_masses);
+    CoarseIsotopePatternGenerator(const Size max_isotope = 0, const bool round_masses = false);
```

Comment:
looks saner.
**TODO: we need to check if there are implications for the python bindings.**